src/less/components/form/TextControl.less

```diff
@@ -1,63 +0,0 @@
-// ===================================== //
```

Comment:
This file - `src/less/components/form/TextControl.less` - refactored in PR #589.